tests/python/gaia-ui-tests/gaiatest/apps/phone/regions/keypad.py

```diff
@@ -126,3 +128,12 @@ def tap_create_new_contact(self):
         new_contact = NewContact(self.marionette)
         new_contact.switch_to_new_contact_form()
         return new_contact
+
+    def tap_add_to_existing_contact(self):
+        add_to_existing_contact = self.marionette.find_element(*self._add_to_existing_contact_locator)
```

Comment:
Yeah, I did that way thinking that maybe that method could be used for other new tests...